.github/workflows/run_tests.yml

```diff
@@ -143,6 +143,13 @@ jobs:
           echo "SMARTSIM_LOG_LEVEL=debug" >> $GITHUB_ENV
           py.test -s --import-mode=importlib -o log_cli=true --cov=$(smart site) --cov-report=xml --cov-config=./tests/test_configs/cov/local_cov.cfg --ignore=tests/full_wlm/ ./tests/backends
 
+      # Run pytest (backends subdirectory)
```

Comment:
Don't forget to update the copypasta'd comment to be correct for this new block (it is not the backends subdirectory).